### ISSUE TITLE
Filter non active players

### ISSUE THREAD

**Issue by marekjagielski** [tomek199/elo-rating#106]
IDEA.
Maybe we don't have to disable non active players. Maybe we could add to Dashboard smth like filter - 'only players that played match last month'. I would be happy to contribute it, however it would be nice to have other opinion.

The problem is that poor active players could loose motivation to play if despite their efforts they are further in ranking then people that hadn't been playing for longer time.

**Comment by tomek199** [comment]
Good idea. We can add come checkbox to Dashboard to see only players that play at least one match last month / last week. Default value of checkbox can be configurable inside league settings component.

**Comment by tomek199** [comment]
Implemented basic case - display in ranking players who played at least one match.

**Comment by tomek199** [comment]
Implemented back-end part - added to player's statistics **lastMatchDate** property
